index.html

```diff
@@ -36,37 +35,39 @@ <h2>O MNĚ</h2>
         </div>
     </section>
 
-    <section class="practice-bem-and-grid">
-        <div class="practice-bem-and-grid__container">
+    <section class="practice">
```

Comment:
Tohle moc chválím! :heart: Krásné, díky barevnému oddělení přesně víš, co se ti kam posouvá a jak se ti to řadí dle použitého gridu. Perfection.